src/languages/php.js

```diff
@@ -18,6 +18,28 @@ export default function(hljs) {
       // Perl at all like $ident$, @ident@, etc.
       `(?![A-Za-z0-9])(?![$])`
   };
+  const CONSTANT = {
+    className: "variable",
+    relevance: 0,
+    variants: [
+      {
+        begin: /const\s*/,
+        end: /\w+/,
+        excludeBegin: true
+      },
+      {
+        begin: /::(?!class\b)/,
```

Comment:
Do you have a specific example of the false positive that cause you to see this?

Reply:
Yes, I added [test](https://github.com/highlightjs/highlight.js/pull/3422/files#diff-a1909111fa7ad598f9584d4ed37e338d8dbd46f522d567017bb237df5c58ae64R8).
Without this regex rule, it would be marked as a variable, instead of a keyword.
```
-        <span class="hljs-keyword">return</span> DateTimeImmutable::<span class="hljs-variable">class</span>;
+        <span class="hljs-keyword">return</span> DateTimeImmutable::<span class="hljs-keyword">class</span>;
```